hil/migrations/versions/76529f0f9e50_add_uuid_and_status_to_networkingaction.py

```diff
@@ -0,0 +1,42 @@
+"""add uuid and status to NetworkingAction
+
+Revision ID: 76529f0f9e50
+Revises: 9089fa811a2b
+Create Date: 2018-01-07 15:24:09.545021
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '76529f0f9e50'
+down_revision = '9089fa811a2b'
+branch_labels = None
+
+# pylint: disable=missing-docstring
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('networking_action', sa.Column('status', sa.String(),
+                  nullable=True))
+    op.add_column('networking_action', sa.Column('uuid', sa.String(),
+                  nullable=True))
+    op.create_index(op.f('ix_networking_action_uuid'), 'networking_action',
+                    ['uuid'], unique=False)
+    op.execute("UPDATE networking_action SET status = 'PENDING', \
+        uuid ='no-uuid'")
```

Comment:
I guess I'd prefer something a bit more explict about why it's not really a uuid. Alternatively, we could go through and generate actual uuids for existing actions. Obviously they wouldn't be usable since no-one would know them, but it avoids having the table in a conceptually odd state untill everything else goes through.

Reply:
Also, I think generating a uuid here would actually fix the migration tests; there's already code in there to normalize things that look like uuids, so as long as we end up with a UUID in this column, it should work.

Reply:
So I did that, and it still doesn't work.
`DETAIL:  Failing row contains (1, null, null, modify_port, 1, 1, vlan/100).`
Something is inserting null for those 2 new columns, although the script is inserting some values (tested this locally).

Another thing, after upgrading I called `hil-admin db downgrade heads`, to see if the downgrade works, but it  actually ran the script for bigInt migration (I think). I am guessing, some changes in downgrade_revision or something somewhere?

If you any other ideas that would be helpful, otherwise ill spend more time digging into the migration stuff.